src/main/java/racinggame/domain/Input.java

```diff
@@ -0,0 +1,61 @@
+package racinggame.domain;
+
+public class Input {
+
+    private static final String SEPARATOR = ",";
+
+    private Input() {}
+
+    public static String[] getCarNames() {
+        String[] carNames;
+
+        while (true) {
+            String inputCarNames = inputCarNames();
+            carNames = splitNameByComma(inputCarNames);
+
+            try {
+                Validator.validatePossibleToStart(carNames);
+                break;
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+
+        return carNames;
+    }
+
+    private static String inputCarNames() {
+        System.out.println("경주할 자동차 이름을 ','로 구분해서 입력하세요.(2대 이상)");
+        return removeBlank(Utils.getInput());
+    }
+
+    private static String[] splitNameByComma(String inputCarNames) {
+        return inputCarNames.split(SEPARATOR);
+    }
+
+    private static String removeBlank(String trial) {
+        return trial.replaceAll("\\s+", "");
```

Comment:
```suggestion
        return trial.replaceAll(SPACE, EMPTY_SPACE);
```
매직넘버를 제거해보는 것은 어떨까요?

Reply:
다른 브랜치에서 반영 되었습니다!!